dist/setup/index.js

```diff
@@ -13359,7 +13359,11 @@ class AdoptOpenJDKDistributor extends base_installer_1.JavaBase {
             const jvmImpl = 'hotspot';
             const versionRange = '[1.0,100.0]';
             const encodedVersionRange = encodeURI(versionRange);
+            let release_type = 'ga';
```

Comment:
nitpick: `release_type` -> `releaseType`
nitpick2: `const releaseType = this.stable ? 'ga' : 'ea'`

Reply:
Done. Thank you.